dask_jobqueue/pbs.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import division
```

Comment:
You can remove all the future imports from this PR because they are in `__init__.py` already.

Reply:
Okay, will look at that and your other comments. Thanks!

Reply:
So I removed import __future__ from __init__.py, and put it in the files where it was needed.